components/infobox/extensions/wikis/stormgate/infobox_extension_attack.lua

```diff
@@ -96,4 +110,15 @@ function Attack._store(data, args, faction, attackIndex)
 	})
 end
 
+
+---@param inputString string?
+---@return string
+function Attack._displayCommaSeparatedString(inputString)
```

Comment:
```suggestion
---@param arr string[]
---@return string
function Attack._displayArray(arr)
	return table.concat(Array.map(arr, function(page)
		return Page.makeInternalLink(value)
	end), ', ')
end
```

Reply:
Is this supposed to substitute the whole `Attack._displayCommaSeparatedString(inputString)`?

Reply:
yeah

Reply:
I don't get it to work for the `data.bonus`, since this is stored differently than `data.targets` and `data.effect` - the others work with data and the new function
![image](https://github.com/user-attachments/assets/895b0980-06ab-4dd6-a77a-046ad7ec80ed)